projects/packages/my-jetpack/_inc/components/product-card/style.module.scss

```diff
@@ -24,11 +24,23 @@
 	display: flex;
 	flex-direction: column;
 
-	&.is-purchase-required,
-	&.plugin_absent {
+	&.is-link {
 		background: none;
 		box-shadow: none;
 		box-shadow: 0 0 0 1px var( --status-plugin_absent ) inset;
+		text-decoration: none;
+		background-color: var( --jp-white-off );
+		color: var( --jp-black-80 );
+
+		&:hover {
+			background-color: var( --jp-white );
+		}
+
+		&:focus {
+			box-shadow: 0 0 0 var( --wp-admin-border-width-focus ) var( --jp-black );
```

Comment:
IMO we should not rely on top of variables that are outside of our control like `--wp-admin-border-width-focus`, if someone changes that there we'll have unexpected behavior. Suggest moving out of it.